includes/class-wc-stripe.php

```diff
@@ -686,11 +686,16 @@ public function add_emails( $email_classes ) {
 		require_once WC_STRIPE_PLUGIN_PATH . '/includes/compat/class-wc-stripe-email-failed-renewal-authentication.php';
 		require_once WC_STRIPE_PLUGIN_PATH . '/includes/compat/class-wc-stripe-email-failed-preorder-authentication.php';
 		require_once WC_STRIPE_PLUGIN_PATH . '/includes/compat/class-wc-stripe-email-failed-authentication-retry.php';
+		require_once WC_STRIPE_PLUGIN_PATH . '/includes/compat/class-wc-stripe-email-failed-refund.php';
+		require_once WC_STRIPE_PLUGIN_PATH . '/includes/compat/class-wc-stripe-email-admin-failed-refund.php';
+		require_once WC_STRIPE_PLUGIN_PATH . '/includes/compat/class-wc-stripe-email-customer-failed-refund.php';
 
 		// Add all emails, generated by the gateway.
 		$email_classes['WC_Stripe_Email_Failed_Renewal_Authentication']  = new WC_Stripe_Email_Failed_Renewal_Authentication( $email_classes );
 		$email_classes['WC_Stripe_Email_Failed_Preorder_Authentication'] = new WC_Stripe_Email_Failed_Preorder_Authentication( $email_classes );
-		$email_classes['WC_Stripe_Email_Failed_Authentication_Retry']    = new WC_Stripe_Email_Failed_Authentication_Retry( $email_classes );
```

Comment:
Nit: we should not remove the `$email_classes` argument unless we document why.

Reply:
We can keep it! We should just have a note like your comment above: the constructor does't accept arguments, so it was already a no-op.

Reply:
I have included it again in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4403/commits/2d975a81c40d5844be3374740b79da5a87506eb7 with a simple comment (I wasn't much sure where to include the comment, let me know if you think it is OK that way).